packages/presentation/lib/utils/extensions/extention_string.dart

```diff
@@ -1,5 +1,15 @@
+import 'package:intl/intl.dart';
+
 extension StrignExtension on String {
   String capitalize() {
     return "${this[0].toUpperCase()}${substring(1).toLowerCase()}";
   }
+
+  String convertData() {
```

Comment:
naming. this name tells me nothing =)

Reply:
changed, but I don't know better naming here
Maybe @alexei-kruk-idf  or @yuliya-klescheva-idf  can help?

Reply:
i would move it in mapper and set name as parseCommentDate